develop/dev-guide-sample-application-nodejs-sequelize.md

```diff
@@ -0,0 +1,323 @@
+</CustomContent>
+<CustomContent platform="tidb-cloud">
+
+**If you don't have a TiDB cluster, you can create one as follows:**
+
+- (Recommended) Follow [Creating a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md) to create your own TiDB Cloud cluster.
+- Follow [Deploy a local test TiDB cluster](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb#deploy-a-local-test-cluster) or [Deploy a production TiDB cluster](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup) to create a local cluster.
+
+</CustomContent>
+
+## Run the sample app to connect to TiDB
+
+This section demonstrates how to run the sample application code and connect to TiDB.
+
+> **Note**
+>
+> For complete code snippets and running instructions, refer to the [tidb-samples/tidb-nodejs-sequelize-quickstart](https://github.com/tidb-samples/tidb-nodejs-sequelize-quickstart) GitHub repository.
+
+### Step 1: Clone the sample app repository
+
+Run the following commands in your terminal window to clone the sample code repository:
+
+```bash
+git clone git@github.com:tidb-samples/tidb-nodejs-sequelize-quickstart.git
+cd tidb-nodejs-sequelize-quickstart
+```
+
+### Step 2: Install dependencies
+
+Run the following command to install the required packages (including `sequelize`) for the sample app:
+
+```bash
+npm install
+```
+
+### Step 3: Configure connection information
+
+Connect to your TiDB cluster depending on the TiDB deployment option you've selected.
+
+<SimpleTab>
+
+<div label="TiDB Serverless">
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+
+2. Click **Connect** in the upper right corner. A connection dialog is displayed.
+
+3. Ensure the configurations in the connection dialog match your operating environment.
+
+    - **Endpoint Type** is set to `Public`
+    - **Connect With** is set to `General`
+    - **Operating System** matches your environment.
+
+    > **Note**
+    >
+    > In Node.js applications, you don't have to provide an SSL CA certificate, because Node.js uses the built-in [Mozilla CA certificate](https://wiki.mozilla.org/CA/Included_Certificates) by default when establishing the TLS (SSL) connection.
+
+4. Click **Create password** to create a random password.
+
+    > **Tip**
+    >
+    > If you have generated a password before, you can either use the original password or click **Reset password** to generate a new one.
+
+5. Run the following command to copy `.env.example` and rename it to `.env`:
+
+    ```shell
+    cp .env.example .env
+    ```
+
+6. Edit the `.env` file, set up the environment variable `DATABASE_URL` as follows, replace the corresponding placeholders `{}` with connection parameters on the connection dialog:
+  
+    ```shell
+    TIDB_HOST='{host}'
+    TIDB_PORT='4000'
+    TIDB_USER='{user}'
+    TIDB_PASSWORD='{password}'
+    TIDB_DB_NAME='test'
+    TIDB_SSL_MODE='true'
```

Comment:
Not recommended to use `SSL_ MODE` as the variable name, is not a bool value in common SSL options.

MySQL:

<img width="1058" alt="image" src="https://github.com/pingcap/docs/assets/5086433/1b7e141b-3c04-41b8-bb74-6c5e5ccf2053">

PG:

<img width="815" alt="image" src="https://github.com/pingcap/docs/assets/5086433/b8b498a2-6fcc-4302-abac-7a415d940aa5">

Reply:
Done, use `TIDB_ENABLE_SSL` instead.

Reply:
Do forget to update the step by step guide.